Engine/ScriptAnalyzer.cs

```diff
@@ -1526,23 +1526,27 @@ public IEnumerable<DiagnosticRecord> AnalyzeScriptDefinition(string scriptDefini
 
             var relevantParseErrors = RemoveTypeNotFoundParseErrors(errors, out List<DiagnosticRecord> diagnosticRecords);
 
-            if (relevantParseErrors != null && relevantParseErrors.Count > 0)
+            int emitParseErrors = severity == null ? 1 : severity.Count(item => item == "ParseError");
```

Comment:
converting this to a boolean and using `Any()` instead of `.Count()` would be better

Reply:
agreed!